### PR TITLE
docs: update installation guide

### PR DESCRIPTION
**Pull request by dejanzele**
This PR...

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Review by olensmar — APPROVED** [review]
thanks!

File: docs/articles/install/3-install-with-helm.md [hunk 1]
```diff
@@ -3,14 +3,32 @@
...
+  * Helm chart - [kubeshop/testkube](https://github.com/kubeshop/helm-charts/tree/main/charts/testkube)
+  * Docker image - [kubeshop/testkube-api-server](https://hub.docker.com/r/kubeshop/testkube-api-server)
+
 ## Prerequisites
 
 Before you proceed with the installation, please ensure that you have the following prerequisites in place:
 
 - Kubernetes cluster (version 1.21+)
 - [Helm](https://helm.sh/docs/intro/quickstart/) (version 3+)
 - [cert-manager](https://cert-manager.io/docs/installation/) (version 1.11+) - Used for TLS certificate management.
```
Comment (ypoplavs) [inline docs/articles/install/3-install-with-helm.md:29]: cert-manager is duplicated twice

File: docs/articles/install/3-install-with-helm.md [hunk 2]
```diff
@@ -77,7 +95,6 @@ By default, the following services will be exposed. You can also choose to overr
 | REST API         | api.$domain        | global.restApiSubdomain        |
 | gRPC API         | agent.$domain      | global.grpcApiSubdomain        |
 | WebSockets API   | websockets.$domain | global.websocketsApiSubdomain  |
-| Status Pages API | status.$domain     | global.statusPagesApiSubdomain |
 | Storage API      | storage.$domain    | global.storageApiSubdomain     |
```
Comment (ypoplavs) [inline docs/articles/install/3-install-with-helm.md:98]: should we also add DEX here?

Comment (dejanzele) [reply docs/articles/install/3-install-with-helm.md:98]: Dex is usually exposes on the API subdomain via path prefix

Comment (ypoplavs) [reply docs/articles/install/3-install-with-helm.md:97]: yeah, but what I meant is that users will see it as one more Ingress in the k8s cluster even though it uses API subdomain

**Review by ypoplavs — APPROVED** [review]

**Pull request merged**